src/main/groovy/com/blackducksoftware/integration/hub/api/vulnerability/VulnerabilityRequestService.java

```diff
@@ -36,7 +37,7 @@ public VulnerabilityRequestService(final RestConnection restConnection) {
     }
 
     public List<VulnerabilityView> getComponentVersionVulnerabilities(final String vulnerabilitiesUrl) throws IntegrationException {
```

Comment:
Might want to think about what it would look like to support two versions of this API, a V1 and a V2 - since a lot of additional data will be available in V2 and then more in V3. 

Might also want to add an issue to add support for V2 (V3 will likely be available as of Hub 4.4).

Reply:
Thanks for the feedback we have two internal tickets to track the changes we need.  One is to address the immediate issue which this code change is for.  We want to release a hotfix release 18.4.1 to fix the immediate concern for Hub 4.3 support when Hub 4.3 releases.  The other internal ticket we have is to address your concerns listed here longer term.